gulpfile.js

```diff
@@ -211,6 +211,41 @@ function clean_cache() {
     return del(['./cache/**'], { force: true });
 };
 
+function yarn_force() {
+    return gulp.src(['./package.json'])
+        .pipe(yarn({ force: true }));
+}
+function yarn_find(done) {
+    var exec = require('child_process').exec;
+    exec('which npm && npm -v && find node_modules -type d', function (err, stdout, stderr) {
```

Comment:
We should really not be using npm here - we have switched to `yarn`.

Reply:
that is part of screwing around trying to figure out what could possibly be wrong on OSX. I was wanting to see if it was explicitly asking for a newer npm, and then somehow its not available on OSX